README.md

```diff
@@ -14,6 +14,129 @@ To run this microservice, you must have SBT installed. You should then be able t
 ### Testing
 ```sbt test it:test```
 
+
+## Endpoints
+
+### GET /returns/vrn/{vrn}?period-key={periodkey}
+
+Where:
+* **vrn** is a valid VRN, for example: "444444444"
+* **periodkey** is the period key of the return, for example: "18AA"
+
+#### Success Response
+
+**HTTP Status**: 200
+
+**Example HTTP Response Body**:
+```
+{
+   "periodKey": "18AA",
+   "vatDueSales": 100.00,
+   "vatDueAcquisitions": 300.00,
+   "vatDueTotal": 400.00,
+   "vatReclaimedCurrPeriod": 150.00,
+   "vatDueNet": 250.00,
+   "totalValueSalesExVAT": 100.00,
+   "totalValuePurchasesExVAT": 100.00,
+   "totalValueGoodsSuppliedExVAT": 100.00,
+   "totalAllAcquisitionsExVAT": 100.00,
+   "agentReferenceNumber": "XAIT0000000000"
+}
+```
+Where:
+* **agentReferenceNumber** is an optional field
+
+#### Error Responses
+
+##### INVALID_VRN
+* **Status**: 400
+
+##### INVALID_PERIODKEY
+* **Status**: 400
+
+##### NOT_FOUND_VRN
+* **Status**: 403
+
+##### INVALID_IDENTIFIER
+* **Status**: 403
+
+##### INVALID_INPUTDATA
+* **Status**: 403
+
+##### NOT_FOUND
+* **Status**: 404
+
+
+### POST /returns/vrn/:vrn
+
+Where:
+* **vrn** is a valid VRN, for example: "444444444"
+
+#### Success Response
+
+**HTTP Status**: 200
+
+**Example HTTP Response Body**:
+```
+{
+   "formBundleNumber": "123456789012"
+}
+```
+
+#### Error Responses
+
+##### INVALID_VRN
+* **Status**: 400
+
+##### INVALID_PAYLOAD
+* **Status**: 400
+
+##### INVALID_SUBMISSION
+* **Status**: 400
+
+##### INVALID_PERIODKEY
+* **Status**: 400
+
+##### DUPLICATE_SUBMISSION
+* **Status**: 409
+
+##### TAX_PERIOD_NOT_ENDED
+* **Status**: 403
+
+##### NOT_FOUND_VRN
+* **Status**: 403
+
+
+### POST /submission
```

Comment:
Can you add an example of what JSON needs sending across for this POST please. Without it the readme doesn't tell people how they could actually use the end point